tests/draft2020-12/optional/format-assertion.json

```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "description": "schema that uses custom metaschema with format-assertion: false",
+        "schema": {
+            "$id": "https://schema/using/format-assertion/false",
+            "$schema": "http://localhost:1234/draft2020-12/format-assertion-false.json",
+            "format": "ipv4"
+        },
+        "tests": [
+            {
+                "description": "format-assertion: false: valid string",
+                "data": "127.0.0.1",
+                "valid": true
+            },
+            {
+                "description": "format-assertion: false: invalid string",
+                "data": "not-an-ipv4",
+                "valid": false
```

Comment:
The results for this test depend on whether the implementation supports the `format-assertion` vocabulary or not. That's fine considering that this is in "optional", but it might be nice to have a `$comment` or something that this is the expected result for an implementation that does support the `format-assertion` vocabulary. Implementations that don't support `format-assertion` will not error, but will return different result.

Reply:
we should probably do that for everything in optional/... I think there's an old issue asking for an expanded README too.

Maybe after the conference is done we can have some discussions about how to improve the layout of the test suite?